scripts/hist_importer.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# This file is part of qutebrowser.
+
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+
+'''Tool to import browser history data from other browsers. Although, safari
+support is still on the way.'''
+
+
+import argparse
+import sqlite3
+import sys
+
+
+def parser():
+    """Parse command line arguments."""
+    description = 'This program is meant to extract browser history from your'\
+                  'previous browser and import them into qutebrowser.'
+    epilog = 'Databases:\n\tQute: Is named "history.sqlite" and can be found '\
+             'at your --basedir. In order to find where your basedir is you '\
+             'can run ":open qute:version" inside qutebrowser.'\
+             '\n\tFirerox: Is named "places.sqlite", and can be found at your'\
+             'system\'s profile folder. Check this link for where it is locat'\
+             'ed: http://kb.mozillazine.org/Profile_folder'\
+             '\n\tChrome: Is named "History", and can be found at the respec'\
+             'tive User Data Directory. Check this link for where it is locat'\
+             'ed: https://chromium.googlesource.com/chromium/src/+/master/'\
+             'docs/user_data_dir.md\n\n'\
+             'Example: $this_script.py -b firefox -s /Firefox/Profile/places.'\
+             'sqlite -d /qutebrowser/data/history.sqlite'
+    parsed = argparse.ArgumentParser(
+        description=description, epilog=epilog,
+        formatter_class=argparse.RawTextHelpFormatter
```

Comment:
Why `argparse.RawTextHelpFormatter`?

Reply:
Because the resulting text gets better(imho) independently of the terminal window's size.
https://i.imgur.com/c9GaDsc.png
vs
https://i.imgur.com/mPk7akK.png